docs/exploration-and-organization/exploration.md

```diff
@@ -6,77 +6,63 @@ redirect_from:
+To view the [models](../data-modeling/models.md) and databases you have access to, click on **Browse data** in the left sidebar.
 
 ![Browse data](./images/browse-data.png)
 
-To learn more, see [Exploring data with Metabase's data browser](https://www.metabase.com/learn/basics/questions/data-browser.html).
-
-## Exploring collections
-
-[Collections][collections] in Metabase are a lot like folders. They're where Metabase keeps all your team's dashboards and charts.
+[Models][models] are tables curated by your team, so they make great starting points for asking questions.
 
-![A collection](./images/collection-detail.png)
+To view the [data reference](./data-model-reference.md) section, click on the **Databases** tab, then click on **Learn about our data** in the upper right. You can also hover over individual tables and click on the book icon.
 
-If your teammates are cool, they'll have pinned some important dashboards or questions within your collections; if so, those pinned items will show up as cards at the top of the collection.
+### Only show verified models
 
-Collections have a list of any other items that are saved within them, and you can see what other collections are saved inside of the current one by checking out the navigation sidebar.
+{% include plans-blockquote.html feature="Only show verified models" %}
 
-## Models
+![Only show verified models](./images/only-verified-models.png)
 
-You can use Models to create derived tables on the fly. See [Models][models].
+One nice-to-have pro feature is that you can filter for verified models by toggling the **Only show verified models** option.
 
-## Exploring dashboards
+## Exploring collections
 
-[Dashboards][dashboards] are a set of questions and text cards that you want to be able to refer to regularly.
+[Collections][collections] in Metabase are a lot like folders. They're where Metabase keeps all your team's dashboards and charts.
 
-If you click on a part of a chart, such as a bar in a bar chart, or a dot on a line chart, you'll see a the **Action menu**, with actions you can take to dive deeper into that result, branch off from it in a different direction, or see automatic insights to [X-ray](x-rays.md) the data.
+![A collection](./images/collection-detail.png)
 
-![Drill-through menu](images/automatic-insights.png)
+If your teammates are cool, they'll have [pinned](./collections.md#pinned-items) some important dashboards or questions within your collections; if so, those pinned items will show up as cards at the top of the collection.
 
-In this example of orders by product category per month, clicking on a dot on this line chart gives us the ability to:
+## Exploring models, questions, and dashboards
```

Comment:
what's the difference between exploring models and browsing models? 
I couldn't see anything explaining models in this paragraph

Reply:
This is an old page (pre-models). Its intent was to show people that you can poke around questions and dashboards as well. 

So: 

- Browsing: Selecting between models. 
- Exploring: drill-through, filtering, summarizing, etc.